.changeset/old-badgers-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'firebase': major
```

Comment:
Changeset should include `@firebase/auth`, in addition to `firebase` - it should have come up as an option when running `yarn changeset`? You can also manually add it.

Reply:
Yes, I just added  `@firebase/auth`.